src/components/ContentSidebar/ActivityFeed/task/Task.js

```diff
@@ -69,6 +68,14 @@ class Task extends React.Component<Props> {
             getMentionWithQuery,
             mentionSelectorContacts,
         } = this.props;
+
+        const isCommentAuthor = currentUser && created_by && created_by.id === currentUser.id;
+
+        const taskPermissions = {
+            canEdit: isCommentAuthor,
```

Comment:
Looks like the comment component looks for `can_edit` and `can_delete`. Also, do you mind updating the flow type in `box-ui-elements.js`? `We never use can_edit_comment` and `can_delete_comment`